### PR TITLE
Prospa Extensions package updates

### PR DESCRIPTION
**Pull request by alhardy**
Related to extensions changes here https://github.com/prospa-group-oss/DotnetExtensions/pull/144

### CODE REVIEW

File: src/ProspaAspNetCoreApiNsb/Controllers/V1/SampleController.cs [hunk 1]
```diff
@@ -29,7 +29,7 @@ public async Task<ActionResult<string>> Get()
             return Ok("sample");
         }
 
-        [Authorize(Constants.Auth.Policies.WritePolicy)]
+        // [Authorize(Constants.Auth.Policies.WritePolicy)]
```
Comment (luizbon) [inline src/ProspaAspNetCoreApiNsb/Controllers/V1/SampleController.cs:32]: Is this intentional or you forgot to clean a test?

Comment (alhardy) [reply src/ProspaAspNetCoreApiNsb/Controllers/V1/SampleController.cs:32]: intentional

Comment (luizbon) [reply src/ProspaAspNetCoreApiNsb/Controllers/V1/SampleController.cs:32]: Why keep commented code then?

Comment (alhardy) [reply src/ProspaAspNetCoreApiNsb/Controllers/V1/SampleController.cs:32]: Just to have the auth example there, this controller will be deleted from a new project anyways, it's there just to have an example controller

**Pull request merged**